### PR TITLE
Visual Feedback for Aiming

### PR DESCRIPTION
**Pull request by joncatanio**
Upon holding down the right trigger the aiming object will change colors gradually. Finally we have some visual feedback on the charged shot. To make it look a bit nicer I increased the max charge shot to 1.5 seconds from 1.0 seconds. This will increase the distance throw by the cookies but it's not by that much and in my opinion is a bit nicer since I had problems hitting houses when stationary before.

The color of the aiming changes to an `Emerald` material but this could be changed. I thought `Ruby` would be nice but red might have the wrong connotation for a positive thing. Let me know what yuh think.

### CODE REVIEW

File: src/CookieActionComponent.cpp [hunk 1]
```diff
@@ -161,21 +162,44 @@ void CookieActionComponent::checkAndPerformAction(double deltaTime, double total
...
+            gameObj3->changeMaterial(materialManager.getMaterial("Brass"));
+            gameObj4->changeMaterial(materialManager.getMaterial("Brass"));
         }
     }
 
+    glm::vec3 aimTarget2 = holder_->getPosition() + throwDirection * 0.5f;
+    gameObj2->setPosition(aimTarget2);
+    if (chargeTime > 0.01 && chargeTime <= 0.3) {
+        gameObj2->changeMaterial(materialManager.getMaterial("Emerald"));
+    }
```
Comment (rgarmsen2295) [inline src/CookieActionComponent.cpp]: Minor nit, but why do these go aimTarget2, aimTarget, aimTarget1, aimTarget3,...?

**Review by rgarmsen2295 — APPROVED** [review]
Code looks good other than minor nit, good stuff!

File: src/CookieActionComponent.cpp (hunk 1 above)
Comment (joncatanio) [reply src/CookieActionComponent.cpp]: Idk actually, @nurgan would know more since it's his code I moved around. So we can see what he has to say.

Comment (joncatanio) [reply src/CookieActionComponent.cpp]: If you're asking about the order lol, it was just an unfortunate ordering issue. I can see if I can adjust it.

Comment (rgarmsen2295) [reply src/CookieActionComponent.cpp]: Ah, fair enough, since that was already there don't let that hold up the PR, not a big deal

Comment (joncatanio) [reply src/CookieActionComponent.cpp]: changed the ordering, but the naming we can discuss with Alex I guess.

**Pull request merged**